README.md

```diff
@@ -2,79 +2,98 @@
 
 ![Microsoft Graph Connect Sample for ASP.NET Core 2.1 screenshot](readme-images/Page1.PNG)
 
-This ASP.NET Core 2.1 MVC sample shows how to connect to Microsoft Graph using delegate permissions and the Azure AD v2.0 endpoint (MSAL) to retrieve a user's profile and profile picture and send an email that contains the photo as an attachment.  
+**Scenario**: Use ASP.NET Core 2.1 MVC to connect to Microsoft Graph using the delegated permissions flow to retrieve a user's profile, their photo from Azure AD (v2.0) endpoint and then send an email that contains the photo as attachment.
+
 The sample uses OpenID Connect for sign in, [Microsoft Authentication Library (MSAL) for .NET](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet) to obtain an access token, and the [Microsoft Graph Client Library for .NET](https://github.com/microsoftgraph/msgraph-sdk-dotnet) (SDK) to interact with Microsoft Graph. The MSAL SDK provides features for working with the [Azure AD v2.0 endpoint](https://azure.microsoft.com/en-us/documentation/articles/active-directory-appmodel-v2-overview), which enables developers to write a single code flow that handles authentication for both work or school (Azure Active Directory) and personal (Microsoft) accounts.
 The sample uses only delegate permissions, therefore it does not require admin consent.
 
->If you are searching for an earlier version of this sample, you can find the ASP.NET Core 1.1 version [here](https://github.com/microsoftgraph/aspnetcore-connect-sample/tree/netcore1.1) and the ASP.NET Core 2.0 version [here](https://github.com/microsoftgraph/aspnetcore-connect-sample/tree/netcore2.0).
+>Previous version of this sample that uses ASP.NET Core 1.1 version is [here](https://github.com/microsoftgraph/aspnetcore-connect-sample/tree/netcore1.1) and the ASP.NET Core 2.0 version [here](https://github.com/microsoftgraph/aspnetcore-connect-sample/tree/netcore2.0).
 
 ## Table of contents
 
-* [Prerequisites](#prerequisites)
-* [Register the app](#register-the-app)
-* [Configure and run the sample](#configure-and-run-the-sample)
-* [Key components of the sample](#key-components-of-the-sample)
-* [Contributing](#contributing)
-* [Questions and comments](#questions-and-comments)
-* [Additional resources](#additional-resources)
-
-## Differences between ADAL and MSAL 
+- [Microsoft Graph Connect Sample for ASP.NET Core 2.1](#microsoft-graph-connect-sample-for-aspnet-core-21)
```

Comment:
Are you sure, you want to include all of these other things in the Table of Contents? I think only the old ones are relevant.

Reply:
@mark-szabo i was trying to make the registration steps in sync with some of our samples. Currently every sample describes the registrations steps differently and its based on who's writing it. I was hoping that we can start to use a start template for registration steps for each app type. 
Do you this is wrong or too much content in the ToC? I don't mind reverting to the old list.

Reply:
Yea, I think if it is long, nobody will read it. I mean, a link to the repo itself in the ToC is not very usable info. Or the link to the ToC in the ToC...

IMHO only the relevant level 2 headers (##) should be included in the ToC.

Reply:
reverted to previous ToC.

Reply:
Thank you @madansr7!